api/src/main/java/io/strimzi/api/kafka/model/listener/KafkaListenerAuthenticationOAuth.java

```diff
@@ -95,8 +93,6 @@ public void setJwksEndpointUri(String jwksEndpointUri) {
     @Description("Configures how often are the JWKS certificates refreshed. " +
             "The refresh interval has to be at least 60 seconds shorter then the expiry interval specified in `jwksExpirySeconds`. " +
             "Defaults to 300 seconds.")
-    @Minimum(1)
```

Comment:
I do not really understand the problem here. With regular YAMLs, when the field is not present, the minimum value is ignored and doesn't cause any issues.